ir/visitor.cpp

```diff
@@ -195,10 +209,12 @@ class Visitor::Tracker {
     struct info_t {
         bool done, visitOnce;
     };
-    typedef std::unordered_map<const IR::Node *, info_t> visited_t;
+    using visited_t = phmap::parallel_flat_hash_map<const IR::Node *, info_t, Util::Hash>;
     visited_t visited;
 
  public:
+    Tracker() : visited(16) {}
```

Comment:
Please add a very brief comment explaining the number 16 🙂 E.g., "pre-allocating 16 slots yields a performance improvement and is sufficient for most cases".

I want to avoid a "huh?" response when people see this code later on.

Reply:
Yeah! Likely we will change with inlined pre-allocated storage in the future, but one step at a time.